articles/ai-services/document-intelligence/faq.yml

```
@@ -468,7 +468,7 @@ sections:
 
             - Advanced
 
-              - **Contributor**: you need this role to create resource group or Document Intelligence resource.
+              - **Contributor**: you need this role to create resource group or Document Intelligence resource.  Please note that having Contributor role does not allow you to list keys for Cognitive Services.  You will still need Cognitive Services User role to use Document Intelligence Studio.
```

```suggestion
              - **Contributor**: you need this role to create a resource group or Document Intelligence resource.  The Contributor role doesn't allow you to list keys for Cognitive Services. To use Document Intelligence Studio, you still need the Cognitive Services User role.
```
Minor edits for Microsoft style.